File: php/src/fizz_buzz.php
<?php

function fizz_buzz(int $n)
{
    switch ([($n % 3 === 0), ($n % 5 === 0)]) {
        case [true, true]:
          return 'Fizz Buzz';
        case [true, false]:
          return 'Fizz';
        case [false, true]:
          return 'Buzz';
        default:
          return $n;
    }
}

File: elixir/lib/fizzbuzz.ex
defmodule Fizzbuzz do
  @moduledoc """
  Documentation for Fizzbuzz.
  """

  def run(num) do
    case { rem(num, 3), rem(num, 5) } do
      {0, 0} -> "Fizz Buzz"
      {0, _} -> "Fizz"
      {_, 0} -> "Buzz"
      _      -> num
    end
  end
end
